services/rds/rds.go

```diff
@@ -302,7 +305,21 @@ func (svc *Service) Discover(ctx context.Context, accessKey, secretKey string) (
 					return errors.WithStack(err)
 				}
 
-				out, err := rds.New(s).DescribeDBInstancesWithContext(ctx, new(rds.DescribeDBInstancesInput))
+				pageNum := 0
+				DBInstanceOutput := []*rds.DescribeDBInstancesOutput{}
+
+				err = rds.New(s).DescribeDBInstancesPagesWithContext(ctx, new(rds.DescribeDBInstancesInput),
+					func(page *rds.DescribeDBInstancesOutput, lastPage bool) bool {
+						pageNum++
+
+						DBInstanceOutput = append(DBInstanceOutput, page)
```

Comment:
Can we collect `page.DBInstances...` instead of `page` so code below will not change?

Reply:
Updated